core/api/api.hpp

```diff
@@ -195,6 +198,52 @@ namespace fc::api {
     IpldObject state;
   };
 
+  struct StartDealParams {
+    DataRef data;
+    Address wallet;
+    Address miner;
+    TokenAmount epoch_price;
+    uint64_t epochs;
```

Comment:
In lotus here are two more fields
MinBlockDuration and DealStartEpoch

As I understood `epochs` is start epoch
Can you add MinBlockDuration?